Model/reviews.post.model.js

```diff
@@ -0,0 +1,47 @@
+const db = require("../db/connection.js");
+
+exports.selectReviewPost = (review_id, patchInfo) => {
+  const toPatchThisKey = Object.keys(patchInfo);
+  const validReviewKeys = [
+    "title",
+    "review_body",
+    "designer",
+    "review_img_url",
+    "votes",
+    "category",
+    "owner",
+  ];
+  return db
+    .query(`SELECT * FROM reviews WHERE review_id = ${review_id.reviews_id}`)
```

Comment:
Also I really dislike this 'review_id.reviews_id' syntax, the naming convention feels flawed. It'd be way better if the parametric endpoint had `:review_id` and you could just destructure that out of req.params in the controller

Reply:
Changed to be more readable